src/test/ui/lint/suggestions.stderr

```diff
@@ -89,12 +89,16 @@ warning: static is marked #[no_mangle], but not exported
    |
 LL |     #[no_mangle] pub static DAUNTLESS: bool = true;
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+   = help: check that enclosing modules are public
```

Comment:
This is hardly a realistic advice, making a module `pub` requires changing the whole crate structure.
More local fix is to export the fn/static from the crate with `pub use`.